src/style.css

```diff
@@ -0,0 +1,129 @@
+.homepage {
+  font-family: Roboto, sans-serif;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --blue-accent: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.homepage__content {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__content {
+  margin: 32px 16px 16px;
+}
+
+.product-card_image {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 36px;
```

Comment:
Margin bottom of image is not same as in the mockup. I did change it to fit all tests.